applications/ISO20022.md

```diff
@@ -0,0 +1,185 @@
+# ISO20022 PoC 
+
+- **Team Name:** Open Smart Contract
+- **Payment Address:** 3GTe5ArJLC8jesr6UFtB9kcoguQmvo8Cuo
```

Comment:
```suggestion
- **Payment Address:** 3GTe5ArJLC8jesr6UFtB9kcoguQmvo8Cuo (BTC)
```

Reply:
Updated in this [commit](https://github.com/OpenSmartContract/W3F-Program/commit/16385bba4c45b5ba0dcd246542d4ba586e5e93c6)